### ISSUE TITLE
[NARRATOR] Narrator does not work for form fields present in 'New Bot Configuration' dialog-box.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#736]
**Actual Result:**
While navigating in 'New Bot Configuration' dialog-box, Narrator focus does not move at various form fields present in this dialog-box and hence Narrator is not able to read about these form-fields.

**Expected Result:**
While navigating in 'New Bot Configuration' dialog-box, Narrator focus should move at various form fields present in this dialog-box and hence Narrator should be able to read about these form-fields.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome screen will open up.
3. Navigate to 'Create a new bot configuration' link and select it.
4. Start Narrator using Ctrl + Windows logo key + Enter.
5. In 'New Bot Configuration' dialog-box, navigate using Tab key and notice that Narrator's focus does not come at various form-fields.

**User Impact:**
Screen reader dependent users won't be able to use the controls which are not receiving screen reader's focus.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge
Tools: Narrator

**Attachments:**
[NARRATOR_BotFrameworkEmulator_NweBotConfiguration _FormFields.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2266117/NARRATOR_BotFrameworkEmulator_NweBotConfiguration._FormFields.zip)

**Comment by tonyanziano** [comment]
Narrator does not work within Electron. I just tested this with NVDA, and all the inputs within the New Bot Configuration modal are labeled and will read out their labels when focused.
